src/main/java/ti4/image/TileHelper.java

```diff
@@ -196,6 +196,11 @@ public static boolean isValidPlanet(String planetID) {
     }
 
     public static Tile getTile(GenericInteractionCreateEvent event, String tileNameOrPos, Game game) {
+        if (tileNameOrPos.matches("^\\s*\\d{3} \\(\\w+\\)\\s*$")) {
+            // If the tileNameOrPos is in the format "123 (XYZ)", we extract the position only
+            tileNameOrPos = tileNameOrPos.trim().substring(0, tileNameOrPos.indexOf(' ')).trim();
+        }
+
```

Comment:
Another user input issue. I run into this sometimes when editing a command on mobile; the verbose autofill input gets broken into actual input text. In that case, just extract the position directly.
- example: https://discord.com/channels/943410040369479690/1341438476528123904/1376710456168681565